src/stories/helpers/add-notes.ts

```diff
@@ -1,5 +1,5 @@
-export default (notes) => {
```

Comment:
definitely a good idea. would this, `export const addNotes = (notes) => {`, be better?